cannelloni.cpp

```diff
@@ -58,8 +58,8 @@ void printUsage() {
   std::cout << "\t\t\t b : enable debugging of interal buffer structures" << std::endl;
   std::cout << "\t\t\t t : enable debugging of interal timers" << std::endl;
   std::cout << "\t -h      \t\t display this help text" << std::endl;
-  std::cout << "Mandatory options:" << std::endl;
   std::cout << "\t -R IP   \t\t remote IP" << std::endl;
+  std::cout << "\t -c seconds \t\t client timeout Second if remote Ip not supplied" << std::endl;
```

Comment:
Also, may be write " -c seconds timeout" and we put a description in the README explaining the option